servers/lib/server/handlers/findteam.coffee

```diff
@@ -24,24 +28,32 @@ module.exports = (req, res) ->
     (user, next) ->
       user.fetchOwnAccount (err, account) ->
         return next err  if err
-        next null, account
+        next null, user, account
 
-    (account, next) ->
+    (user, account, next) ->
       account.fetchRelativeGroups (err, groups) ->
-        next err, account, groups
+        next err, user, account, groups
 
-    (account, groups, next) ->
+    (user, account, groups, next) ->
       groups = groups.filter (group) -> group.slug isnt 'koding'
 
+      if not groups.length
+        return next(
+          if user.lastLoginDate < FAREWELL_SOLO_DATE
```

Comment:
@sinan, here I detect old solo user checking if they don't have any group and haven't logged in since the day when solo login was closed (it's a date of solo login blocking changes deployment - 07/19/16). If it's a solo user, they are redirected to farewell-solo page.
However, I'm not sure this check is enough. Maybe additionally we need to check that user is not blocked in any group?